notification-consumer-services/src/main/java/uk/ac/bbsrc/tgac/miso/notification/consumer/service/mechanism/IlluminaNotificationMessageConsumerMechanism.java

```diff
@@ -147,18 +149,23 @@ public void setAttemptRunPopulation(boolean attemptRunPopulation) {
             log.debug("Saving new run and status: " + runName);
             if (!run.has(IlluminaTransformer.JSON_STATUS)) {
               // probably MiSeq
-              r = new IlluminaRun();
+              // https://jira.oicr.on.ca/browse/GLT-1611
+              // Changed back to IlluminaRun() in future.
+              r = new RunImpl();
               r.setPlatformRunId(Integer.parseInt(m.group(2)));
               r.setAlias(runName);
               r.setFilePath(runName);
               r.setDescription(m.group(3));
               r.setPairedEnd(false);
               is.setHealth(ht);
               r.setStatus(is);
+              r.setPlatformType(PlatformType.ILLUMINA);
             } else {
               String xml = run.getString(IlluminaTransformer.JSON_STATUS);
-              is = new IlluminaStatus(xml);
```

Comment:
It's done in ```((RunImpl) r).illuminaRunConfig(xml, null);```.   That's on line189 of ```RunImpl.java```.

I think a reference of this new instance of IlluminaStatus will need be exposed because it's referenced later in ```IlluminaNotificationMessageConsumerMechanism.java```.